### ISSUE TITLE
Add support for patient vital signs measurements

### ISSUE THREAD

**Issue by qligier** [hl7ch/ch-emed#148]
Body weight and height may be needed for the drug dosing.

Should the CH Core Patient be extended and those Observations be added in the contained elements?

Example of body weight coding: https://hl7.org/fhir/observation-example.html

**Comment by oliveregger** [comment]
FHIR defines profiles for vitalsigns, e.g. bodyweight is here: http://hl7.org/fhir/StructureDefinition/bodyweight

instead of adding it as contained i would suggest  to add them as full resource to the bundle (the observation points to the patient), and maybe add a vital section in the composition to reference it, contained should be only used for resources which have no own existence.

**Comment by ziegm** [comment]
Should we add a vitalSigns section in all documents?
* e.g. Profile [Medication Card Composition](http://build.fhir.org/ig/hl7ch/ch-emed/branches/148_vitalsigns/StructureDefinition-ch-emed-composition-medicationcard.html) with additional section according to [IPS](http://hl7.org/fhir/uv/ips/StructureDefinition-Composition-uv-ips.html)
   * Examples [Composition](http://build.fhir.org/ig/hl7ch/ch-emed/branches/148_vitalsigns/Composition-MedicationCard-VitalSigns.html) with [Body Height](http://build.fhir.org/ig/hl7ch/ch-emed/branches/148_vitalsigns/Observation-BodyHeight.html) and [Body Weight](https://build.fhir.org/ig/hl7ch/ch-emed/branches/148_vitalsigns/Observation-BodyWeight.html)
* TBD: title 'Vital Signs' in DE/FR/IT

**Comment by pjolo** [comment]
Feedback from IPAG:
- Would like to leave the eMedication as defined in the IPAG report.
- The information on height and weight should be available in the primary system and does not need to be shown in the eMedication.
- Experience should first be gained with the minimal dataset
- If there are many requests after the establishment, it can be discussed again.

Conclusion:
Vital signs should not be included in the eMedication documents from IPAG's point of view.

**Comment by qligier** [comment]
I agree that it doesn't have to be added to the minimal dataset yet, but it doesn't cost much to add it to the specifications as a "may support" already.
We have found the patient weight multiple times in our prescription samples, so it seems more important than thought by IPAG.

**Comment by pjolo** [comment]
The request will not be implemented as no new information should be included as defined in the IPAG report. The issue will be kept open for future discussion.
